src/explore-education-statistics-common/src/components/form/FormCheckboxGroup.tsx

```diff
@@ -151,6 +154,9 @@ export const BaseFormCheckboxGroup = ({
           underline={false}
         >
           {selectAllText(isAllChecked, options)}
+          {visuallyHiddenText && (
+            <VisuallyHidden>{visuallyHiddenText}</VisuallyHidden>
```

Comment:
Can you add a space before `{visuallyHiddenText}` to make sure the words don't run together.